### ISSUE TITLE
Extension viewlet icons are blurry

### ISSUE THREAD

**Issue by Tyriar** [microsoft/vscode#124601]
Notice how much AA is happening on the edges of the image:

![image](https://user-images.githubusercontent.com/2193314/119537870-59ac8580-bd3f-11eb-9dfd-7a5459928ad9.png)

This is happening because this image is loaded for the bottom one for example https://tyriar.gallerycdn.vsassets.io/extensions/tyriar/luna-paint/0.5.1/1620472207047/Microsoft.VisualStudio.Services.Icons.Default which is 256x256 but it's scaled down to 42x42 (when devicePixelRatio=1), causing the excessive blurriness.

**Comment by Tyriar** [comment]
Actually it occurs to me that this is because the icon is provided at 256x256 as recommended by some webpage (though I'm seeing 128x128 is recommended at https://code.visualstudio.com/api/working-with-extensions/publishing-extension#marketplace-integration).

It's also called out in https://code.visualstudio.com/api/working-with-extensions/publishing-extension#marketplace-integration that svg don't work, these icons could be much more crisp if they were svgs. An alternative solution would be to allow providing multiple extension images at different dimensions.

**Comment by misolori** [comment]
I don't think that having the right sized images would make a difference here? [Check out this sample I created](https://codepen.io/miguelsolorio/full/PopjjqX), left column has individually sized icons and right size has the largest image resized.

**Comment by Tyriar** [comment]
Huge difference on standard dpi:

![image](https://user-images.githubusercontent.com/2193314/119545300-84024100-bd47-11eb-98df-567ec558ad5e.png)
![image](https://user-images.githubusercontent.com/2193314/119545339-911f3000-bd47-11eb-8e08-67d4fd435d1d.png)

**Comment by misolori** [comment]
😮 Well I was wrong about that. Then I think we'd need to have different icons coming from the marketplace and then only display the correct ones depending on the screen resolution via [`srcset`](https://caniuse.com/?search=srcset):

```html
<img src="icon-44.png" srcset="icon-44@2x.png">
```

**Comment by Tyriar** [comment]
Or an svg, that's probably what most icons would be derived from anyway.

**Comment by misolori** [comment]
Yes, SVG would be best but I think there were some security limitations from the marketplace that have prevented this.

**Comment by sandy081** [comment]
Closing this as out of scope because I do not see any action can be taken unless MP supports svgs.
